src/main/java/ca/uhn/fhir/jpa/starter/CustomConsentService.java

```diff
@@ -132,4 +137,36 @@ private String getPatientFromToken(RequestDetails theRequestDetails) {
     String patRefId = oAuth2Helper.getPatientReferenceFromToken(jwt, OAUTH_CLAIM_NAME);
     return patRefId;
   }
+
+  /*
+   * For Patch Request we take Request body(byte array) and convert to json string Then using
+   * ObjectMapper we can convert json string to List of key value pairs(map object)
+   */
+  @SuppressWarnings("unchecked")
```

Comment:
What cast is this for?

Reply:
@hankwallace  This is suggested by sonar lint. It is for converting using ObjectMapper